core/bcast/bcast.go

```diff
@@ -63,13 +65,27 @@ func (b Broadcaster) Broadcast(ctx context.Context, duty core.Duty,
 			return err
 		}
 
-		return b.eth2Cl.SubmitAttestations(ctx, []*eth2p0.Attestation{att})
+		err = b.eth2Cl.SubmitAttestations(ctx, []*eth2p0.Attestation{att})
+		if err != nil {
+			log.Error(ctx, "Failed to submit attestation to beacon node", err, z.Int("slot", int(duty.Slot)))
```

Comment:
as per go guidelines, do not log AND return errors

Reply:
yeah you are right, we shouldn't log and return errors. but i added this to log so that it will be visible in logs at the time running charon-docker-compose or individual node.

Although, since we are returning error it will get propagated between core components